### DISCUSSION TITLE
10.0.1 board finally assembled ...

### DISCUSSION THREAD

**Discussion by rszemeti** [peterantypas/maiana#125] [General]
Can I assume that the 11.x.x 422 binaries are going to work on this without much issue?

I'm just about to pop it in the oven to bake ... I noticed the 470uH inductor in the 10.1 BOM is listed as 0603 when its really 1206 on the board ... and the i2C EEPROM is missing from the BOM, but I had one in stock, so its ready to cook!

I just need to remember how to load the bootloader now ...

**Comment by peterantypas** [comment]
I'm not sure. 10.x is 2 years old already and I don't even have support for
that board in FW. You may have to tweak the project.

**Reply by rszemeti** [reply]
Yep, understood.  Having bought several iterations of bare PCB and nearly got around to assembling them before the next one was released, I eventually had to draw a line in the sand and go "I'm building this one"

I did at one point know how to build and install the bootloader, hopefully I can figure that out again.  I have a vague recollection of CubeIDE being involved ...

**Reply by rszemeti** [reply]
That seems to have gone OK, I got the transponder firmware to and bootloader firmware to build in Eclipse CDT,  comparing bsp_10_0.hpp and bsp_11_3.hpp it looks like no pins have moved around, so (apart from the crystal freq issue which I guess will depend on my individual xtals, no problem I can check tx freq on counter/analyser) I think it should "work" ...

I built the "422 legacy" variant as I've not ordered an adaptor board yet ... 

I get some serial data out of it:

$PAISYS,11.x,4.1.0,,STM32L422,0,0*51
$PAISTN,0,,,0,0,0,0,0*11
$PAITXCFG,1,1,1,0,0*0B

So its doing *something* at least ...  I think the biggest difference id the polarity of the GPS_EN signal, I'll fiddle with it and see what happens.

**Comment by rszemeti** [comment]
I have components for 4 more boards ... I may end up ordering some 11.x boards to add to my every growing collection of "boards I didn't assemble" :)  

My first ones were the "stacked" boards, then the "single board" version, then the 10.0.1 ... it looks like the BOM is very similar so only a few extra bits to buy ... and this project has done wonders for my stock of surface mount inductors, I'll never run out!
